### DISCUSSION TITLE
docker - acme.sh command only causes load

### DISCUSSION THREAD

**Discussion by kw-sudo** [acmesh-official/acme.sh#4994] [Q&A]
Hi,

I'm running acme.sh as a docker container on my Synology NAS. If I only start a terminal command `acme.sh` or `acme.sh --help`, the cursor is blinking and nothing happens. After a few seconds CPU and Memory load runs up until the Diskstation freezes.

Installed via task planer:

- [ ] docker run -d --name acme.sh \
- [ ] -v /volume1/docker/acme:/acme.sh \
- [ ] --net=host \
- [ ] --restart always \
- [ ] neilpang/acme.sh:latest daemon

What I found out:

If I remove the volume enviroment it work's. I've checked all required ownerships, they're correct. In terminal I can check nslookup, dns settings, cc., but no `acme.sh`.

Connecting via ssh terminal@root with `docker run --rm  -it neilpang/acme.sh /bin/sh`, I get a prompt and commands are working (bridge mode, no volumes). Even there, set a volume `/docker/acme:/acme.sh` leads to the same result.

I stuck since an announced update inside docker to version 3.0.8. Register a new container with separated versions down to 3.0.0 were not successful.

Further informations:
- Synology Diskstation DS716+II
- DSM 7.2.1-69057 Update 4
- Docker version 20.10.23, build 876964a
- Linux Diskstation 3.10.108 #69057
- acme.sh 3.0.8

Any ideas?

Regards,
Oliver

**Comment by rioncm** [comment]
You objective of running acme.sh is unclear. Are you trying to get a cert for the NAS or something else? As I understand it acme.sh is not a standalone application, it is a utility that needs to be pointed at a solution. 

Check out [Synology DSM 7 with Let’s Encrypt and DNS Challenge](https://dr-b.io/post/Synology-DSM-7-with-Lets-Encrypt-and-DNS-Challenge) for an implementation concept on the NAS.

**Reply by kw-sudo** [reply]
I have completely lost sight of the issue and have now solved the certificate via docker@RasPi and nginx ProxyManager.
